packages/terra-button/src/Button.module.scss

```diff
@@ -106,26 +106,26 @@
   }
 
   .text-only {
-    margin-left: var(--terra-button-text-only-horizontal-margin, 0.714rem);
-    margin-right: var(--terra-button-text-only-horizontal-margin, 0.714rem);
+    margin-left: var(--terra-button-text-only-horizontal-margin-left, 0.714rem);
+    margin-right: var(--terra-button-text-only-horizontal-margin-right, 0.714rem);
   }
 
   .icon-only {
-    margin-left: var(--terra-button-icon-only-horizontal-margin, 0.5rem);
-    margin-right: var(--terra-button-icon-only-horizontal-margin, 0.5rem);
+    margin-left: var(--terra-button-icon-only-horizontal-margin-left, 0.5rem);
+    margin-right: var(--terra-button-icon-only-horizontal-margin-right, 0.5rem);
```

Comment:
We can drop the word horizontal from the variable name

Reply:
https://github.com/cerner/terra-core/pull/2125/commits/8341e6dfa00e294efaeff9b272fbbb6f3a15e7c9